package.json

```diff
@@ -391,7 +391,8 @@
           "description": "Specifies Lint tool name.",
           "enum": [
             "golint",
-            "gometalinter"
+            "gometalinter",
+            "megacheck"
```

Comment:
I tried testing this by declaring but not using a variable, expecting the linting error "declared but not used".

golint and gometalinter send such warnings via stdout which is what the extension uses.

but megacheck seems to be sending the warnings via stderr. which is why they don't show up in the Problems view.

Can you confirm?